opensrp-stock/src/main/java/org/smartregister/stock/domain/StockResponse.java

```diff
@@ -0,0 +1,13 @@
+package org.smartregister.stock.domain;
+
+import java.io.Serializable;
+import java.util.List;
+
+public class StockResponse implements Serializable {
```

Comment:
Why do we need this class

Reply:
I used it together with `gson` to convert the JsonObject response to a Pojo